test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -400,3 +417,80 @@ def test_conv2d_serials_binary_unary_with_quantizer_api(self):
                 node_occurrence,
                 node_list,
             )
+
+    @skipIfNoX86
+    def test_linear_with_quantizer_api(self):
+        """
+        Test pattern of single linear with X86InductorQuantizer.
+        """
+        with override_quantized_engine("x86"), torch.no_grad():
+            for use_bias in [True, False]:
+                m = TestHelperModules.SingleLinearModule(use_bias).eval()
+                example_inputs = (torch.randn(2, 4),)
+                quantizer = X86InductorQuantizer().set_global(
+                    xiq.get_default_x86_inductor_quantization_config()
+                )
+                node_occurrence = {
+                    # one for input and weight, one for output
+                    torch.ops.quantized_decomposed.quantize_per_tensor.default: 2,
+                    torch.ops.quantized_decomposed.dequantize_per_tensor.default: 2,
+                    torch.ops.quantized_decomposed.quantize_per_channel.default: 1,
+                    torch.ops.quantized_decomposed.dequantize_per_channel.default: 1,
+                }
+                node_list = [
+                    torch.ops.quantized_decomposed.quantize_per_tensor.default,
+                    torch.ops.quantized_decomposed.dequantize_per_tensor.default,
+                    torch.ops.aten.addmm.default if use_bias else torch.ops.aten.mm.default,
```

Comment:
if you don't need to land this before branch cut, I think you could switch to `capture_pre_autograd_graph` for capture and you will see a single aten.linear op instead

Reply:
Hi @jerryzh168. Thanks for the suggestion. We hope to land this before code freeze. We feel difficult to switch to new API as code freeze is near. Is it OK to land this first then we switch to new API later?

Reply:
sure, that's fine